ndmg/scripts/ndmg_bids.py

```diff
@@ -57,7 +57,9 @@
 #
 # *these files can be anywhere up stream of the dwi data, and are inherited.
 skippers = ['slab907', 'slab1068', 'DS01216', 'DS01876',
-            'DS03231', 'DS06481', 'DS16784', 'DS72784']
+            'DS03231', 'DS06481', 'DS16784', 'DS72784',
+            'hemispheric_res-1x1x1', 'hemispheric_res-2x2x2',
+            'tissue_res-1x1x1', 'tissue_res-2x2x2']
```

Comment:
My previous comment was directed at this -- inside the `for` loop for the group level analysis should be a `try`/`except` block where we fail gracefully for arbitrary parcellations that may have isolates.